lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -88,6 +94,10 @@ def configure(conf)
       @remove_keys_on_update = @remove_keys_on_update.split ','
     end
 
+    if @logstash_prefix_key && @logstash_prefix_key.is_a?(String)
```

Comment:
Thanks @danmcp, I kept the same pattern as the previous code to avoid being different.  If it doesn't hurt anything, I was thinking of leaving it.